doc/release-notes-1.2.0.rst

```diff
@@ -0,0 +1,586 @@
+--------------------------------
+
+* :ref:`asset_tracker`:
+
+  * Added functionality to configure high/low thresholds for sensor data, so that only data below/above the threshold is sent to the cloud.
+  * Modified the command format to match the format that is used by nRF Cloud.
+  * Implemented support for receiving modem AT commands from the cloud and returning the modem's response.
+  * Added functionality to configure the interval at which sensor data is sent to the cloud.
+    This makes it possible to change the poll/send interval for environmental and light sensors from the terminal card in nRF Cloud.
+  * Replaced ``printk`` calls with calls to the :ref:`zephyr:logger` subsystem.
+  * Added a separate workqueue for the application, instead of using the system workqueue.
+
+* :ref:`gps_with_supl_support_sample`:
+
+  * Added support for the new :ref:`supl_client` library, if enabled.
+
+
+Updated libraries
+-----------------
+
+* :ref:`lib_download_client`:
+
+  * Added the :option:`CONFIG_DOWNLOAD_CLIENT_MAX_TLS_FRAGMENT_SIZE` option that allows to configure fragment sizes for TLS connections and non-TLS connections independently.
+  * Added support for using non-default ports.
+
+* :ref:`lib_spm`:
+
+  * Updated the security attribution to configure the peripherals NRF_REGULATORS and NRF_WDT as Non-secure.
+  * Added the RTC0 peripheral (as Non-Secure).
+  * Fixed a bug where the library attempted to set the IRQ target state of the P0 peripheral.
+
+* :ref:`lib_fota_download`:
+
+  * Added an optional progress event (:cpp:enumerator:`FOTA_DOWNLOAD_EVT_PROGRESS <fota_download::FOTA_DOWNLOAD_EVT_PROGRESS>`) that informs the user of the library how many bytes have been downloaded.
+  * Fixed a bug where the library continued downloading even if writing to the DFU target failed.
+  * Implemented a mechanism to retry downloads if a socket error occurs.
+
+* :ref:`lib_aws_fota`:
+
+  * Added functionality to resume jobs that are marked as being in progress, which ensures a more robust FOTA operation through AWS IoT jobs.
+  * Added offset reporting through the ``statusDetails`` field in an AWS IoT job, which makes it possible to track the progress of a FOTA operation more precisely.
+  * Removed the unused ``app_version`` parameter from the :cpp:func:`aws_fota_init` function.
+  * Inversed the interpretation of the return value of :cpp:func:`aws_fota_mqtt_evt_handler`.
+    0 now indicates success, and no further handling is required.
+    1 indicates that further processing is required by the :cpp:func:`mqtt_evt_handler` that called :cpp:func:`aws_fota_mqtt_evt_handler`.
+
+* :ref:`lib_nrf_cloud`:
+
+  * Removed the button/switch pairing method.
+  * Added functionality to handle the device configuration in the device shadow.
+
+* :ref:`liblwm2m_carrier_readme`:
+
+  * Updated to version 0.8.1.
+
+* Moved the following libraries from ``drivers/`` to ``lib/``:
+
+  * :ref:`at_cmd_readme`
+  * ``lte_link_control``
+
```

Comment:
The AT host library has changes default line ending from `CR` to `LF` to support sending SMS using the AT host